modules/op-setting-up-openshift-pipelines-to-download-or-view-sboms.adoc

```diff
@@ -0,0 +1,181 @@
+    task.results.format: application/text
+    task.results.key: LINK_TO_SBOM  # <.>
+    task.results.type: external-link  # <.>
+spec:
+  results:
+    - description: Contains the SBOM link  # <.>
+      name: LINK_TO_SBOM
+  steps:
+    - name: print-sbom-results
+      image: quay.io/image  # <.>
+      script: | # <.>
+        #!/bin/sh
+        syft version
+        syft quay.io/<username>/quarkus-demo:v2 --output cyclonedx-json=sbom-image.json
+        echo 'BEGIN SBOM'
+        cat sbom-image.json
+        echo 'END SBOM'
+        echo 'quay.io/user/workloads/<namespace>/node-express/node-express:build-8e536-1692702836' | tee $(results.LINK_TO_SBOM.path) # <.>
+----
+<.> The name of your task.
+<.> The location for storing the task outputs.
+<.> The SBOM task result name. Do not change the name of the SBOM result task.
+<.> (Optional) Set to open the SBOM in a new tab.
+<.> The description of the result.
+<.> The image that generates the SBOM.
+<.> The script that generates the SBOM image.
+<.> The SBOM image along with the path name.
+
+. Update the Pipeline to reference the newly created SBOM task.
+
++
+[source,yaml]
+----
+...
+spec:
+  tasks:
+    - name: sbom-task  
+      taskRef: 
+        name: sbom-task # <.>
+  results:
+    - name: IMAGE_URL  # <.>
+      description: url
+      value: $(tasks.sbom-task.results.LINK_TO_SBOM) # <.>
+----
+<.> The same name as created in Step 2.
+<.> The name of the result.
+<.> The OCI image repository URL which contains the `.sbom` images.
+
+. Rerun the affected Openshift Pipeline.
+
+
+== Viewing an SBOM in the web UI
+
+.Prerequisites
+
+* You set up Openshift Pipelines to download or view SBOMs.
+
+.Procedure
+
+. Navigate to the Activity -> PipelineRuns tab.
+
+. For the component whose SBOM you want to view, select its most recent pipeline run.
+
+. On the PipelineRun details page, select *View SBOM*.
+
+.. You can use your web browser to immediately search the SBOM for terms that indicate vulnerabilities in your software supply chain. For example, try searching for `log4j`.
+
+.. You can select *Download* to download the SBOM, or *Expand* to view it full-screen.
+
+== Downloading an SBOM in the CLI
+
+.Prerequisites
+
+* Install the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/Secure-your-supply-chain/proc_inspect_sbom/[Cosign] CLI tool.
+
+* You set up Openshift Pipelines to download or view SBOMs.
+
+.Procedure
+
+. Open terminal, log in from any perspective, and then switch to the relevant project.
+
+. Use Cosign to download the SBOM. From the output of the last command, pass the image path as an argument into Cosign’s `download sbom` command. Be sure to delete any quotation marks around the image path.
++
+Example
++
+[source,terminal]
+----
+$ cosign download sbom quay.io/<workspace>/user-workload@sha256:<output omitted>"
+----
+
+.. (Optional) To view the full SBOM in a searchable format, you can redirect the output:
```

Comment:
"you can" should probably be removed

Reply:
Also "you can" is not needed while the explanation of the command is needed. "(Optional) To view the full SBOM in a searchable format, redirect the output by entering the following command:" or something similar, in my view